emc-lite/emc-lite.cxx

```diff
@@ -45,14 +49,16 @@ class emclite : public PhysicsModel {
     kappa_e0 = options["kappa_e0"].doc("parallel conduction coefficient").withDefault(1.0);
     kappa_epar = kappa_e0 * pow(Te, 5 / 2);
 
-    bool parallel_sheaths = options["parallel_sheaths"].doc("parallel boundary conditions").withDefault(false);
+    parallel_sheaths = options["parallel_sheaths"]
+                           .doc("parallel boundary conditions")
+                           .withDefault(false);
     Nt = options["Nt"].doc("Density at target").withDefault(1.0);
     Tt = options["Tt"].doc("Temperature at target").withDefault(20.0);
-    Cs = options["Cs"].doc("Sound speed").withDefault(1.0);
+    Cs0 = options["Cs"].doc("Sound speed").withDefault(1.0);
```

Comment:
perhaps we keep the input consistent with the code? Input should then read "Cs0".